src/components/Hearing.js

```diff
@@ -104,6 +113,18 @@ export class Hearing extends React.Component {
     );
   }
 
+  getManageButton() {
```

Comment:
That would actually be much more straight forward. And if we don't need to keep "toolbarless" hearing view at all, we could just use the `HearingView` to handle read, edit, create use cases. That could simplify things in the long run.